docs/devel/testing.md

```diff
@@ -174,55 +174,7 @@ hack/test-integration.sh  # Run all integration tests.
 
 ## End-to-End tests
 
-* e2e tests build kubernetes and deploy a cluster of nodes.
-  - Generally on a specific cloud provider.
-* Access gcr.io images
-* Access a specific, non-latest image tag (unless testing pulling).
-* Tests may not flake due to intermittent issues.
-* Use ginko to desribe steps.
-  - See [should run a job to completion when tasks succeed](../../test/e2e/job.go)
-* Use [NewDefaultFramework](../../test/e2e/framework.go)
-  - Contains clients, namespace and auto resource cleanup
-* See [coding conventions](coding-conventions.md).
-
-### e2e test philosophy
```

Comment:
Primarily to DRY up the docs, so that future updates around e2e testing happen in one place vs. two places.  My read of the e2e-tests doc was that all this info was sufficiently expressed there.  Maybe that doc lacks something that's as tl;dr-ish as this?